src/components/ADempiere/DefaultTable/CellInfo.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <el-tag
+    v-if="isDocumentStatus"
+    key="document-status"
+    :type="documentStatus"
+    disable-transitions
+  >
+    {{ displayedValue }}
+  </el-tag>
+
+  <span v-else key="info-value">
+    {{ displayedValue }}
+  </span>
+</template>
+
+<script>
+import { defineComponent, computed } from '@vue/composition-api'
+
+import { COLUMNS_NAME_DOCUMENT_STATUS, FIELDS_DECIMALS } from '@/utils/ADempiere/references.js'
+import { typeValue } from '@/utils/ADempiere/valueUtils.js'
+import { formatField, convertBooleanToTranslationLang } from '@/utils/ADempiere/valueFormat.js'
+
+export default defineComponent({
+  name: 'CellInfo',
+
+  props: {
+    fieldAttributes: {
+      type: Object,
+      required: true
+    },
+    dataRow: {
+      type: Object,
+      default: () => {}
+    }
+  },
+
+  setup(props, { root }) {
+    const { columnName } = props.fieldAttributes
+    const fieldValue = props.dataRow[columnName]
+
+    const isDocumentStatus = computed(() => {
+      return COLUMNS_NAME_DOCUMENT_STATUS.includes(columnName)
+    })
+
+    const displayedValue = computed(() => {
+      return formatValue(props.dataRow, props.fieldAttributes)
+    })
+
+    const documentStatus = computed(() => {
+      return root.tagStatus(fieldValue)
+    })
+
+    const formatNumber = ({ displayType, number }) => {
+      let fixed = 0
+      // Amount, Costs+Prices, Number
+      if (FIELDS_DECIMALS.includes(displayType)) {
+        fixed = 2
+      }
+      return new Intl.NumberFormat().format(number.toFixed(fixed))
+    }
```

Comment:
Already exists valuefornat

Reply:
changed to use `formatPrice` and `formatQuantity`. We have to improve the number formatting functionality because if the value has only zero in decimals it will format it as integer (`$ 765` should be `$ 765.00`), also if it is empty it doesn't format in zero but it returns empty value (empty value should be => `$ 0.00`).